scripts/ec2cleanup.py

```diff
@@ -129,7 +132,34 @@ def delete_unused_network_interfaces(provider_mgmt, excluded_enis, output):
         logger.exception('Exception in %r', delete_unused_network_interfaces.__name__)
 
 
-def ec2cleanup(exclude_volumes, exclude_eips, exclude_elbs, exclude_enis, output):
+def delete_stacks(provider_mgmt, excluded_stacks, output):
+    stack_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for stack in provider_mgmt.list_stacks():
+            if excluded_stacks and stack.stack_name in excluded_stacks:
+                logger.info("  Excluding Stack name: %r", stack.stack_name)
```

Comment:
OPTIONAL: preceding space

Reply:
I think there are intentional for formatting

Reply:
yes. that's it